geodiff/src/drivers/postgresdriver.cpp

```diff
@@ -471,15 +471,16 @@ static bool isColumnInt( const TableColumnInfo &col )
 
 static bool isColumnDouble( const TableColumnInfo &col )
 {
-  return col.type == "real" || col.type == "double precision";
+  return col.type == "real" || col.type == "double precision" || startsWith( col.type.dbType, "numeric" );
```

Comment:
shall we also add "decimal" type ? postgres docs say it should be equivalent to "numeric" https://www.postgresql.org/docs/9.1/datatype-numeric.html